pages/open-reading-frames/[uuid].js

```diff
@@ -63,30 +70,38 @@ export default function OpenReadingFrame({ orf, isJson }) {
                   </DataItemValue>
                 </>
               )}
-              {orf.pct_identical_protein && (
+              {orf.pct_identical_protein >= 0 && (
```

Comment:
We have a `truthyOrZero` function in lib/general.js to do this, but I’m thinking your way here is easier to understand. So let’s leave this as is. I might convert other things from `truthyOrZero` to this over time.